src/View/ThemeResourceLoader.php

```diff
@@ -162,101 +170,6 @@ public function getPath($identifier)
         return Path::normalise($modulePath . $subpath, true);
     }
 
-    /**
-     * Attempts to find possible candidate templates from a set of template
-     * names from modules, current theme directory and finally the application
-     * folder.
-     *
-     * The template names can be passed in as plain strings, or be in the
-     * format "type/name", where type is the type of template to search for
-     * (e.g. Includes, Layout).
-     *
-     * The results of this method will be cached for future use.
-     *
-     * @param string|array $template Template name, or template spec in array format with the keys
-     * 'type' (type string) and 'templates' (template hierarchy in order of precedence).
-     * If 'templates' is omitted then any other item in the array will be treated as the template
-     * list, or list of templates each in the array spec given.
-     * Templates with an .ss extension will be treated as file paths, and will bypass
-     * theme-coupled resolution.
-     * @param array $themes List of themes to use to resolve themes. Defaults to {@see SSViewer::get_themes()}
-     * @return string Absolute path to resolved template file, or null if not resolved.
-     * File location will be in the format themes/<theme>/templates/<directories>/<type>/<basename>.ss
-     * Note that type (e.g. 'Layout') is not the root level directory under 'templates'.
-     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it.
-     */
-    public function findTemplate($template, $themes = null)
```

Comment:
We never deal with absolute template file paths anymore except inside template engines themselves.